src/styles/main.scss

```diff
@@ -1,3 +1,74 @@
+$start-day: (
+  mon 0,
+  tue 101px,
+  wed 202px,
+  thu 303px,
+  fri 404px,
+  sat 505px,
+  sun 606px
+);
+
+@mixin squareSize($sizeOfdays) {
+  width: $sizeOfdays;
+  height: $sizeOfdays;
+}
+
+%positionCenter {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
 body {
   margin: 0;
 }
+
+.page {
+  @extend %positionCenter;
+  min-height: 100vh;
+}
+
+.calendar {
+  max-width: 720px;
```

Comment:
`max-width` is not the best choice here

<img width="561" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/1983879/d92bf0c6-8cd6-4534-a64a-0c724305a4a4">